identity-mapping.json

```diff
@@ -69,14 +69,59 @@
 		"locationHierarchyForProfiling": {
 			"value": "zone,postalCode"
 		},
-		"addressLine1": {
-			"value": "addressLine1"
+		"presentAddressLine1": {
+			"value": "presentAddressLine1"
 		},
-		"addressLine2": {
-			"value": "addressLine2"
+		"presentAddressLine2": {
+			"value": "presentAddressLine2"
 		},
-		"addressLine3": {
-			"value": "addressLine3"
+		"presentAddressLine3": {
+			"value": "presentAddressLine3"
+		},
+		"presentAddressLine4": {
+			"value": "presentAddressLine4"
+		},
+		"permanentAddressLine1": {
+			"value": "permanentAddressLine1"
+		},
+		"permanentAddressLine2": {
+			"value": "permanentAddressLine2"
+		},
+		"permanentAddressLine3": {
+			"value": "permanentAddressLine3"
+		},
+		"permanentAddressLine4": {
+			"value": "permanentAddressLine4"
+		},
+		"permanentCountry": {
+			"value": "permanentCountry"
+		},
+		"permanentProvince": {
+			"value": "permanentProvince"
+		},
+		"permanentCity": {
+			"value": "permanentCity"
+		},
+		"permanentBarangay": {
+			"value": "permanentBarangay"
+		},
+		"permanentZipcode": {
+			"value": "permanentZipcode"
+		},
+		"presentCountry": {
+			"value": "presentCountry"
+		},
+		"presentProvince": {
+			"value": "presentProvince"
+		},
+		"presentCity": {
+			"value": "presentCity"
+		},
+		"presentBarangay": {
+			"value": "presentBarangay"
+		},
+		"presentZipcode": {
+			"value": "presentZipcode"
```

Comment:
Do we need any all these new values ? Which code is using all these ?

Reply:
The automation team is using these values while activating UIN in /identity API.

Reply:
If we are adding this only for automation tools compatibility, I would like to understnad this from the automation team better.